### DISCUSSION TITLE
Allow to change the default font size in editor.

### DISCUSSION THREAD

**Discussion by ayophanz** [WordPress/gutenberg#43522] [Q&A] [answered]
## What problem does this address?

In gutenberg Editor it would be nice to have capability to change the default font size. 

## What is your proposed solution?

It would be great to `add_action` or `add_filter` in functions.php for this small enhancement.

**Comment by t-hamano** [comment, marked as answer]
Thank you for submitting the issue.
You can change the default font size in the following way:

## On Classic Theme

Register a style sheet describing the following style via `add_editor_style()`.

```css
body {
  font-size: 20px;
}
```

## On Block Theme

You can configure this in theme.json.

```json
{
  "version": 2,
  "styles": {
    "typography": {
      "fontSize": "20px"
    }
  }
}
```

**Comment by Mamaduka** [comment]
I think this issue is more suitable for [Discussions](https://github.com/WordPress/gutenberg/discussions).

**Reply by t-hamano** [reply]
Thank you!
I didn't know that issues can be converted to Discussions 😅

**Reply by Mamaduka** [reply]
You can find action in the issue sidebar.

**Comment by ayophanz** [comment]
Thank you all
